test/SubscriptionExample.Tests/SubscriptionExample.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
Whitespaces instead of tabs.

Reply:
Auto-format due to missing .editorconfig.  Another PR.